src/components/data-list/index.tsx

```diff
@@ -84,6 +100,21 @@ export const DataList = ({
       );
       setTotalResultsCount(headers.get(totalCountHeaderName));
       setModelData(data);
+      localStorage.setItem(
```

Comment:
@gsplakmal how is it supposed to work when you have multiple views/grids for different entities? Will corresponding settings for different views override each other?

Reply:
This is fixed with unique keys for storage